cmds/exp/uefiboot/main.go

```diff
@@ -2,6 +2,19 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
+// uefiboot loads the firmware generated by EDK2's UefiPayLoadPkg to boot a UEFI based OS
+//
+// Synopsis:
+//     uefiboot [ARGS] payload
+//
+// Options:
+//     -d: print debug output
+//     -i n: Address where to load the payload image
+//     -serial_addr  n: serial IO port address
+//     -serial_width n: serial port register width
+//     -serial_hertz n: serial port input hertz
```

Comment:
hertz? I had no idea :-)

Reply:
No thinking went into this PR, I just copied the description of the parameters down below ;)